torchtune/modules/loss/ppo.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional, Tuple
+
+import torch
+import torch.nn as nn
+from torchtune.utils import ppo_utils
+
+
+class PPOLoss(nn.Module):
+    """
+    Proximal Policy Optimization (PPO) Loss module: https://arxiv.org/abs/1707.06347.
+
+    Using:
+        https://github.com/vwxyzjn/lm-human-preference-details/blob/ccc19538e817e98a60d3253242ac15e2a562cb49/lm_human_preference_details/train_policy_accelerate.py#L719
+        https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974143998/baselines/ppo2/model.py#L68-L75
+
+    as references.
+    Args:
+        gamma (float): Discount factor.
+        lmbda (float): lambda parameter for GAE.
+        epsilon (float): clipping parameter for PPO.
+    """
+
+    def __init__(
+        self,
+        gamma: float = 1.0,
+        lmbda: float = 0.95,
+        epsilon: float = 1e-5,
```

Comment:
Why such a small default value for epsilon? In this case won't we always just clamp the ratio to almost identically 1?

Reply:
yeah, eps is usually 0.1/0.2